src/OpenTelemetry/Logs/ILogger/OpenTelemetryLoggingExtensions.cs

```diff
@@ -75,19 +173,19 @@ public static class OpenTelemetryLoggingExtensions
 
         var services = builder.Services;
 
-        if (configureOptions != null)
-        {
-            // TODO: Move this below the RegisterLoggerProviderOptions call so
-            // that user-supplied delegate fires AFTER the options are bound to
-            // Logging:OpenTelemetry configuration.
-            services.Configure(configureOptions);
-        }
-
         // Note: This will bind logger options element (eg "Logging:OpenTelemetry") to OpenTelemetryLoggerOptions
         RegisterLoggerProviderOptions(services);
 
         services.AddOpenTelemetrySharedProviderBuilderServices();
 
+        if (configureOptions != null)
```

Comment:
Do we need a CHANGELOG entry for this change?

Reply:
Added